math/mathcore/inc/Math/Math_vectypes.hxx

```diff
@@ -0,0 +1,5 @@
+#include "RConfigure.h"
+#ifdef R__HAS_VECCORE
+#include <VecCore/VecCore>
+using Double_v = typename vecCore::backend::VcVector::Double_v;
```

Comment:
Make senses ... but we now are trying to be less invasive :).   So for new typedef, please consider using ROOT::Double_v .... otherwise this will leading to odd/weird conflict for other project with their own Double_v (Geant, VecGeom, etc.)